### PR TITLE
SGC-2079 : [Coddestyle] Add readme for Java and JS

### PR DESCRIPTION
**Pull request by Serykh92**
### Why?

A description of the project and instructions were needed

### What's Changing?

 Add README for project and installation instructions Java. I also added a file that describes the contribution rules

### CODE REVIEW

File: .github/CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+ 
+## Pull Request Process  
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.  
+2. Update the README.md with details of changes  
+3. Pull requests must be merged only by Kaseya developers   
+## Code of Conduct
```
Comment (aglumova) [inline .github/CONTRIBUTORS.md:8]: Please add new line before title.

File: java/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+3. ### Add dependency in your pom.xml or build.gradle
+* For Maven
+```
+<dependency>
+  <groupId>com.kaseya</groupId>
+  <artifactId>static-analyzers</artifactId>
+  <version>20220826114552</version>
+</dependency>
+```
+* For Gradle
```
Comment (aglumova) [inline java/README.md:65]: Please add new empty line.

File: .github/CONTRIBUTORS.md (hunk 1 above)
Comment (Serykh92) [reply .github/CONTRIBUTORS.md:9]: fixed

File: java/README.md (hunk 2 above)
Comment (Serykh92) [reply java/README.md:69]: fixed

**Pull request merged**